### DISCUSSION TITLE
리트코드 스터디 4주차 모임

### DISCUSSION THREAD

**Discussion by SamTheKorean** [DaleStudy/leetcode-study#411] [공지사항]
스터디 2기 참여자 여러분, 안녕하세요. Sam입니다👋

네 번째 모임도 전 모임과 같이 지원자를 받고 각각 문제에 대해서 설명 및 논의하는 시간을 가지겠습니다!

문제당 한 분씩 풀이를 할 예정이고, 신청은 선착순으로 각 주차에 있는 아직 풀이 희망자가 없는 문제 중 원하시는 문제와 디스코드 명을 one-bit-integer / Sam 과 같은 형식으로 댓글로 남겨주시면 됩니다!ㅎㅎ

다음 모임도 이전과 마찬가지로 아래 링크로 RSVP 부탁드리겠습니다! (https://discord.gg/5N4BSC6M?event=1279818287105970176)

**Comment by TonyKim9401** [comment]
world-search / Tony

**Reply by TonyKim9401** [reply]
발표용 PPT 입니다.
https://docs.google.com/presentation/d/1SX-6aHEDWPuobSN6wfWDJq3h03mqAYvgRBJU2qFkksA/edit#slide=id.g28760b2257c_1_17

**Comment by wogha95** [comment]
missing-number / wogha95

**Comment by jdalma** [comment]
longest-consecutive-sequence / 정현준

**Reply by DaleSeo** [reply]
https://github.com/DaleStudy/leetcode-study/pull/422#discussion_r1746508148

**Reply by jdalma** [reply]
제가 일요일 오전에 일이 생겨서 이번 모임은 참여가 힘들 것 같습니다 ㅠㅜ

**Reply by DaleSeo** [reply]
@jdalma 알겠습니다! 미리 알려주셔서 감사헤요.

**Comment by highballplz** [comment]
maximum-product-subarray / highball
